pipeline/psth.py

```diff
@@ -448,11 +446,10 @@ def make(self, key):
         behav_lr = {k: np.where(behav['trial_instruction'] == k) for k in lr}
 
         try:
-            egpos = (ephys.ElectrodeGroup.ElectrodeGroupPosition()
-                     & key).fetch1()
+            egpos = (ephys.ProbeInsertion.InsertionLocation * experiment.BrainLocation & key).fetch1()
         except dj.DataJointError as e:
             if 'exactly one tuple' in repr(e):
```

Comment:
hm.. perhaps we should subclass errors to avoid these types of tests. It is a dubious practice to rely on the error message to be fixed.

Reply:
This was discussed at one point in `https://github.com/datajoint/datajoint-python/issues/540`, closed due to non-specificity of having a general issue. We should probably define a mechanism to track broad-stroke changes such as this.